src/Uno.UI/UI/Xaml/Controls/Image/ImageSizeHelper.cs

```diff
@@ -9,7 +9,8 @@ namespace Windows.UI.Xaml.Controls
 {
 	internal static class ImageSizeHelper
 	{
-		public static void MeasureSource(this Image image, Size finalSize, ref Rect child)
+		[Pure]
+		public static Size MeasureSource(this Image image, Size finalSize, Size child)
```

Comment:
'child' is a confusing parameter name for a `Size`

Reply:
I agree it could be improved.